lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -16,10 +17,9 @@ def initialize(serializer, options = {})
         end
 
         def serializable_hash(options = nil)
-          options = {}
```

Comment:
I needed to use `options` on https://github.com/bacarini/active_model_serializers/blob/master/lib/active_model/serializer/adapter/json_api.rb#L31